ui/src/react-admin/modules/content-page/components/ContentEditForm/ContentEditForm.tsx

```diff
@@ -152,9 +152,9 @@ export const ContentEditForm: FunctionComponent<ContentEditFormProps> = ({
 
 	// Render
 	const owner: PickerItem | undefined = {
-		label: contentPageInfo.owner.fullName,
+		label: contentPageInfo.owner?.fullName,
 		type: ContentPickerType.PROFILE,
-		value: contentPageInfo.owner.id,
+		value: contentPageInfo.owner?.id,
```

Comment:
if we can't find the owner id, because owner is undefined, shouldn't we set the whole owner object to undefined instead of {label: undefined, type: profile, value: undefined} ?